3-platform/warehouse/spec/src/main/java/com/_6core/platform/warehouse/rest/v1/inventory/Inventory.java

```diff
@@ -0,0 +1,10 @@
+package com._6core.platform.warehouse.rest.v1.inventory;
+
+public record Inventory(
+        String inventoryId,
+        String warehouseId,
+        String productId,
+        Integer actualQuantity,
+        Integer availableQuantity
```

Comment:
actual and available quantities are our system implementation.
user should know nothing about the system implementation. 
```suggestion
        Integer quantity
```